test/js/bun/http/bun-server.test.ts

```diff
@@ -407,6 +407,25 @@ describe("Server", () => {
       const result = await fetch(`https://${url}`, { tls: { rejectUnauthorized: false } }).then(res => res.text());
       expect(result).toBe("Hello");
     }
+
+    // Test that HTTPS keep-alive doesn't cause it to re-use the connection on
+    // the next attempt, when the next attempt has reject unauthorized enabled
+    {
+      expect(
+        async () => await fetch(`https://${url}`, { tls: { rejectUnauthorized: true } }).then(res => res.text()),
```

Comment:
```suggestion
        async () => await fetch(server.url, { tls: { rejectUnauthorized: true } }).then(res => res.text()),
```

Reply:
we probably should refactor to `const url = server.url.href;` that already includes the protocol (the same for the other places in this test)